### PR TITLE
Enable retester for openshift-knative org

### PR DESCRIPTION
**Pull request by pierDipi**
Signed-off-by: Pierangelo Di Pilato <pierdipi@redhat.com>

### CODE REVIEW

**Review by matzew — COMMENTED** [review]
Love it!

LGTM, thanks for driving this @pierDipi

File: clusters/app.ci/prow/03_deployment/retester.yaml [hunk 1]
```diff
@@ -38,6 +38,7 @@ spec:
         - --job-config-path=/etc/job-config
         - --cache-file=/cache/backoff
         - --enable-on-org=openshift
+        - --enable-on-org=openshift-knative
```
Comment (jmguzik) [inline clusters/app.ci/prow/03_deployment/retester.yaml:41]: @hongkailiu -> our new retester. Are we ready to accept more orgs or should we suggest using the old one in this case?

**Review by jmguzik — COMMENTED** [review]
/cc @hongkailiu

File: clusters/app.ci/prow/03_deployment/retester.yaml (hunk 1 above)
Comment (hongkailiu) [reply clusters/app.ci/prow/03_deployment/retester.yaml:41]: Yes. Let us do it.

**Review by hongkailiu — APPROVED** [review]

**Pull request merged**